doc/whats_new/v1.5.rst

```diff
@@ -43,6 +43,9 @@ Security
 Changed models
 --------------
 
+- |Efficiency| The `clustering.MeanShift` class has now improved computational speed as it properly converges for constant data.
+  :pr:`28951` by :user:`Akihiro Kuno <akikuno>`.
+
```

Comment:
I would consider it a bug and move that in the `cluster` section of the changelog.